### PR TITLE
LPS 131395 Fix line chart and pie chart popover styles

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
Pie chart and line chart popover styles in Content Performance panel are not in sync with Lexicon popover styles
[Link to ticket](https://issues.liferay.com/browse/LPS-131395)

Proposed Solution
========
We have added .popover classes to charts popovers so that they inherit the styles from Clay Popovers. We have also removed unnecessary styles.

Steps to Verify:
----------------
1. Open Content Perfomance Panel
2. Hover over the donut chart
3. Hover over the line chart
4. If you want to verify this behaviour with more data cherry-pick this commit: https://github.com/claraizquierdo/liferay-portal/commit/e3275f1cc0a127c46bfaf623aadafc9e968c1245

Change summary (optional):
---------------
* LPS-131395 Change pie chart popover styles to be the same as ClayPopover
* LPS-131395 Change line chart popover styles to be the same as ClayPopover

Screenshots (optional):
-----------------------
Before:
unstlyed-tooltip

After:
popover with modified spacing

### CODE REVIEW

**Comment by claraizquierdo** [comment]
ci:test:sf

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/analytics-reports-app.scss [hunk 1]
```diff
@@ -25,11 +25,7 @@ html#{$cadmin-selector} {
 			}
 
 			.custom-tooltip {
-				background-color: #fff;
-				border: 1px solid #ccc;
-				margin: 0;
-				opacity: 0.9;
-				padding: 4px;
+				position: static;
```
[the remark this answers is not in the document]
Comment (claraizquierdo) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/analytics-reports-app.scss:28]: I understand your curiosity: .popover class gives us a `position:absolute` and, I don't know why Recharts didn't show the popover when it has absolute position so I had to change it to static. 🤷‍♀️

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:forward

**Review by Stephy6 — COMMENTED** [review]
Review finished!

**Pull request closed without merging**